demo/api/src/auth/access-control.service.ts

```diff
@@ -7,15 +7,21 @@ export class AccessControlService extends AbstractAccessControlService {
         if (user.isAdmin) {
             return UserPermissions.allPermissions;
         } else {
-            const deniedPermissions = ["userPermissions"];
-            return availablePermissions.filter((permission) => !deniedPermissions.includes(permission)).map((permission) => ({ permission }));
+            const deniedPermissions = ["userPermissions", "news"];
+            return [
+                ...availablePermissions.filter((permission) => !deniedPermissions.includes(permission)).map((permission) => ({ permission })),
+                { permission: "news", contentScopes: [{ domain: "main", language: "en" }] },
+            ];
```

Comment:
Filtering availablePermissions here has the downside of having to exclude permissions that are only available for a specific scope.